docs/guides/blocks/formulas.md

```diff
@@ -30,20 +32,74 @@ Formulas can accept parameters, which is the primary way for them to access data
 
 ## Results
 
-TODO
+All formulas must return a result, which is a single value matching the type specified in the `resultType` property of the formula definition. See the [Data types guide][data-types] for more information on the type of values that can be returned.
+
+
+## Authentication
+
+Formulas can use [authentication][authentication] to fetch private data. When using [system authentication][system_auth] there is no change to the user experience, but when you use [user authentication][user_auth] the formula editor will be automatically updated to prompt for a connected account. The connected account will appear as the first parameter of the formula, but it's value is not passed to your formula's `execute` method.
```

Comment:
Nit, "its" in "it's value"

Reply:
You can tell by now this is my weak spot 😄.  Fixed.